docs/architecture.md

```diff
@@ -36,17 +36,17 @@ with a short description and the main Perseus data type it accepts.
 </thead>
 <tbody>
     <tr>
-        <td>`ServerItemRenderer`</td>
-        <td>`PerseusItem`</td>
+        <td><code>ServerItemRenderer</code></td>
```

Comment:
I'm sure you know this, but MD supports tables.

| My    | Header   |
| ----- | -------- |
| A     | Cell     |
| Some  | `Code`   |

```
| My    | Header   |
| ----- | -------- |
| A     | Cell     |
| Some  | `Code`   |
```

Reply:
I do!  :)  But I found writing long-form content in a MD table extremely annoying as you can't "line-wrap" cells afaik. I also found that the MD preview in VSCode doesn't handle code backticks in tables. Maybe those are both weak arguments, but I hope everyone working in Perseus understands HTML... 😂 

## Example (works, but is clunky to edit). 

| My | Header | Abc | 
| -- | ------ | --- | 
| `ServerItemRenderer` | `PerseusItem` | The main exercise renderer. This component is a convenience wrapper around the `Renderer`. It adds some management of displaying hints and showing/hiding a keypad (if one is available). |